Letterbook.Docs/_pages/deploy.md

```diff
@@ -0,0 +1,5 @@
+---
+title: Deploy Letterbook
+---
+
+Something about docker compose
```

Comment:
tbh, this is far enough in the future that it's probably best to just not mention it yet.

I expect docker compose will be one option, but not the one I would recommend to most people. It's not all that difficult to host apt and rpm repositories, and I expect to make those the preferred install and deploy path.